client/src/app/pages/applications/applications-table-assessment/applications-table-assessment.tsx

```diff
@@ -544,12 +555,9 @@ export const ApplicationsTable: React.FC = () => {
         <Toolbar {...toolbarProps}>
           <ToolbarContent>
             <ToolbarBulkSelector
-              onSelectAll={selectAll}
-              areAllSelected={areAllSelected}
-              selectedRows={selectedItems}
+              {...toolbarBulkSelectorProps}
               paginationProps={paginationProps}
               currentPageItems={currentPageItems}
```

Comment:
You can also remove `paginationProps` and `currentPageItems` from here, `toolbarBulkSelectorProps` includes them.